extensions/expression-language/tests/ExpressionLanguageTests.cpp

```diff
@@ -187,13 +195,13 @@ TEST_CASE("GetFile PutFile dynamic attribute", "[expressionLanguageTestGetFilePu
   auto repo = std::make_shared<TestRepository>();
 
   std::string in_dir("/tmp/gt.XXXXXX");
-  REQUIRE(testController.createTempDirectory(&in_dir[0]) != nullptr);
+  REQUIRE(!testController.createTempDirectory(&in_dir[0]).empty());
```

Comment:
This doesn't serve a great deal of purpose. we should be using the temp dirs not just throwing them away.